### PR TITLE
Add meeting_user collection and actions and tests for it.

### PR DESCRIPTION
**Pull request by reiterl**
First mini step to remove the template fields. Allow to move the fields to the user_meeting action.

### CODE REVIEW

File: global/meta/models.yml [hunk 1]
```diff
@@ -1181,6 +1204,10 @@ meeting:
     restriction_mode: B
 
   # Users
+  user_meeting_ids:
+    type: relation-list
+    to: user_meeting/meeting_id
+    restriction_mode: TODO
```
Comment (r-peschke) [inline global/meta/models.yml:1210]: B

File: openslides_backend/action/actions/__init__.py [hunk 2]
```diff
@@ -40,6 +40,7 @@ def prepare_actions_map() -> None:
         theme,
         topic,
         user,
+        user_meeting,
```
Comment (r-peschke) [inline openslides_backend/action/actions/__init__.py:43]: s.o. name

File: global/meta/models.yml [hunk 3]
```diff
@@ -302,6 +302,11 @@ user:
     to: committee/forwarding_user_id
     restriction_mode: E
 
+  user_meeting_ids:
+    type: relation-list
+    to: user_meeting/user_id
+    restriction_mode: TODO
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml:308]: Done.

File: global/meta/models.yml [hunk 4]
```diff
@@ -302,6 +302,11 @@ user:
     to: committee/forwarding_user_id
     restriction_mode: E
 
+  user_meeting_ids:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml]: Renamed.

File: global/meta/models.yml [hunk 5]
```diff
@@ -302,6 +302,11 @@ user:
...
+    type: relation-list
+    to: user_meeting/user_id
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml]: Renamed.

File: global/meta/models.yml [hunk 6]
```diff
@@ -452,6 +457,24 @@ user:
     required: True
     restriction_mode: F
 
+user_meeting:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply global/meta/models.yml]: Renamed.

File: global/meta/models.yml (hunk 1 above)
Comment (reiterl) [reply global/meta/models.yml:1210]: Done.

File: openslides_backend/action/actions/__init__.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/__init__.py]: Done.

**Review by r-peschke — COMMENTED** [review]
So far LGTM

**Pull request merged**